### PR TITLE
added 4 projects and deleted the ones that no longer work

### LINKED ISSUE #504: Update-final-projects

### PR DESCRIPTION
**Pull request by Arfah**
#closes #504 i've updated the projects and deleted ones that don't work. Please squirrel!

### CODE REVIEW

**Comment by Arfah** [comment]
FYI.. hold on from squirrelling just yet as I need to change something.

**Comment by ptolemybarnes** [comment]
@Arfah Is this blocked?

**Comment by Arfah** [comment]
@ptolemybarnes no longer blocked. Please squirrel!

File: data/graduate_projects.yml [hunk 1]
```diff
@@ -1,3 +1,35 @@
+- name: Changr
+  team: Hamza Sheikh, Mahmud Hussain,Fergus Lemon, Samuel Overloop and Radu Asandei.
+  cohort: October 2015
+  blurb:  Changr is an app to help the less advantaged in society to receive donations on the fly using Bluetooth Beacons.
```
Comment (ptolemybarnes) [inline data/graduate_projects.yml]: I think this should be:
"Changr is an app to help the less advantaged in society receive donations"
Can't really explain why.

**Comment by ptolemybarnes** [comment]
It's not part of this pull request but just noticed this:
image

*what

**Comment by ptolemybarnes** [comment]
Couple of edits to make and then :shipit: :)

**Pull request merged**